server/src/capture.rs

```diff
@@ -0,0 +1,198 @@
+use log::{error, info};
```

Comment:
Add docs -- a title

Other fixes:
1. Sort includes by origin (std lib, 3rd party, local).
2. Use `cargo fmt` and `cargo clippy --tests` to fix lints.